rfcs/20220713-sig-open-xla.md

```diff
@@ -0,0 +1,82 @@
+# RFC: Establish SIG OpenXLA
+
+| Status        | Proposed     |
+:-------------- |:---------------------------------------------------- |
+| **RFC #**     | [419](https://github.com/tensorflow/community/pull/419)|
+| **Author(s)** | Thea Lamkin (thealamkin@google.com), Mehdi Amini (aminim@google.com) |
+| **Sponsor**   | Thea Lamkin (thealamkin@google.com)            |
+| **Updated**   | 2022-07-13                                           |
+
+## Objective
+
+OpenXLA will be a community-driven and modular open source compiler. It will enable efficient lowering, optimization and deployment of ML models from most major frameworks to any hardware backend notably CPUs, GPUs, and ML ASICs. This work will be done collaboratively with major ML frameworks and hardware vendors.
+
+SIG OpenXLA will focus on creating the OpenXLA project, including the extraction of XLA from TensorFlow into a standalone project. SIG discussions will facilitate coordination around roadmap, design evolution, and new workflows to be created in OpenXLA. 
+
+## SIG Charter
+
+### Goals
+
+* Accelerate industry collaboration around XLA and build a vibrant OSS community.
+* Share and receive feedback on the technical direction for OpenXLA and ensure it meets the needs of major users and contributors.
+* Set up a new XLA repository or organization with independent build/test, with infra to more easily accept PRs, and that is hardware and framework independent. 
```

Comment:
I suggest to evaluate the pros and cons of using an independent Github org also related to the Keras migration experience.

One of the main issue:
- No ticket migration [between different Github orgs](https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository)

Reply:
What is this all bout